File: Java/Kode Program/Pokemon.java
public class Pokemon extends Kartu {
    private String id;
    private String namaKartu;
    private String serial;

    public Pokemon() {}

    public Pokemon(String id, String namaKartu, String serial, String namaPermainan, String bahan) {
        super(namaPermainan, bahan);
        this.id = id;
        this.namaKartu = namaKartu;
        this.serial = serial;
    }

    public void setID(String i) { this.id = i; }
    public void setNamaKartu(String nk) { this.namaKartu = nk; }
    public void setSerial(String s) { this.serial = s; }

    public String getID() { return id; }
    public String getNamaKartu() { return namaKartu; }
    public String getSerial() { return serial; }

    @Override
    public void printInfo() {
        System.out.println("Pokemon ID: " + id + ", Nama Kartu: " + namaKartu + ", Serial: " + serial);
        super.printInfo();
    }
}

File: Python/Kode Program/Pokemon.py
from Kartu import Kartu

class Pokemon(Kartu):
    def __init__(self, id="", namaKartu="", serial="", namaPermainan="", bahan=""):
        super().__init__(namaPermainan, bahan)
        self.id = id
        self.namaKartu = namaKartu
        self.serial = serial

    def setID(self, i): self.id = i
    def setNamaKartu(self, nk): self.namaKartu = nk
    def setSerial(self, s): self.serial = s

    def getID(self): return self.id
    def getNamaKartu(self): return self.namaKartu
    def getSerial(self): return self.serial

    def printInfo(self):
        print(f"Pokemon ID: {self.id}, Nama Kartu: {self.namaKartu}, Serial: {self.serial}")
        super().printInfo()
